src/testbed_nerf.cu

```diff
@@ -938,6 +940,11 @@ __global__ void composite_kernel_nerf(
 			rgb = Array3f::Constant(cam_fwd.dot(pos - origin) * depth_scale);
 		} else if (render_mode == ERenderMode::AO) {
 			rgb = Array3f::Constant(alpha);
+		} else if (render_mode == ERenderMode::Confidence) {
```

Comment:
maybe add comment explaining that the definition of confidence is still in progress